classes/user/PKPUserAction.inc.php

```diff
@@ -117,8 +118,7 @@ public function mergeUsers($oldUserId, $newUserId)
             }
         }
 
-        $userDao = DAORegistry::getDAO('UserDAO'); /** @var UserDAO $userDao */
-        $userDao->deleteUserById($oldUserId);
+        Repo::user()->delete(Repo::user()->get($oldUserId));
 
         return true;
     }
```

Comment:
I'd like to see thee `PKPUserAction` methods merged into the Repository and then removed.

Reply:
This is split between PKPUserAction and UserAction; what's your opinion on extending PKP\user\Repository into APP\user\Repository and injecting that? Still the go-to for cases like this?

Reply:
Done, thanks -- OJS and OMP required an extended Repository class, OPS didn't.